src/main/java/net/snowflake/ingest/SimpleIngestManager.java

```diff
@@ -588,12 +590,13 @@ public HistoryRangeResponse getHistoryRange(
       requestId = UUID.randomUUID();
     }
 
-    try (CloseableHttpResponse response =
-        httpClient.execute(
-            builder.generateHistoryRangeRequest(
-                requestId, pipe, startTimeInclusive, endTimeExclusive))) {
+    HttpGet request =
+        builder.generateHistoryRangeRequest(requestId, pipe, startTimeInclusive, endTimeExclusive);
```

Comment:
For my own reference, why do we need to move this out?

Reply:
The request would be used twice later at [here](https://github.com/snowflakedb/snowflake-ingest-java/blob/6927aaaf0b68cfa6c1e40df28512a535d0f6a661/src/main/java/net/snowflake/ingest/SimpleIngestManager.java#L596) and [here](https://github.com/snowflakedb/snowflake-ingest-java/blob/6927aaaf0b68cfa6c1e40df28512a535d0f6a661/src/main/java/net/snowflake/ingest/SimpleIngestManager.java#L599). Moved it out side to avoid building it twice.